src/physics/euler.cpp

```diff
@@ -192,6 +192,29 @@ ::convert_primitive_to_conservative ( const std::array<real,nstate> &primitive_s
     return conservative_soln;
 }
 
+template <int dim, int nstate, typename real>
+inline std::array<real,nstate> Euler<dim,nstate,real>
+::convert_conservative_to_entropy ( const std::array<real,nstate> &conservative_soln ) const
+{
+    const real pressure = compute_pressure<real>(conservative_soln);
+    const real density = conservative_soln[0];
+    const real vel2 = compute_velocity_squared(compute_velocities(conservative_soln));
+    // This is a different entropy than the "entropy measure" in compute_entropy_measure()
+    // Therefore, has been hard-coded
```

Comment:
can delete this, i think its clear enough that they're difference

Reply:
okay, deleted